src/rpdk/core/jsonutils/resolver.py

```diff
@@ -116,6 +122,10 @@ def _get_array_container_type(property_schema):
             return ContainerType.LIST
         return ContainerType.SET
 
+    @staticmethod
+    def _get_multiple_lang_type():
+        return ResolvedType(ContainerType.MULTIPLE, MULTIPLE)
```

Comment:
It's better to pass the multiple types information, `return ResolvedType(ContainerType.MULTIPLE, schema_type)`